src/snowflake/snowpark/session.py

```diff
@@ -21,6 +21,7 @@
 import pkg_resources
 
 from snowflake.connector import ProgrammingError, SnowflakeConnection
+from snowflake.connector.config_manager import _get_default_connection_params
```

Comment:
Will this import work in stored proc?

Reply:
Sorry I didn't catch your message here 🤦 yea it won't work actually.... I'll put out another PR to fix it